specs/test_formats/README.md

```diff
@@ -75,7 +84,13 @@ The second is still somewhat ambiguous: some tests may want cover multiple forks
 There is a common factor here however: the options are exclusive, and give a clear idea on what test suites need to be ran to cover testing for a specific fork.
 The way this list of forks is interpreted, is up to the test-runner:
 State-transition test suites may want to just declare forks that are being covered in the test suite,
- whereas shuffling test suites may want to declare a list of forks to test the shuffling algorithm for individually. 
+ whereas shuffling test suites may want to declare a list of forks to test the shuffling algorithm for individually.
+
+Test-formats specify the following `forks` interpretation rules:
+
+- `collective`: the test suite applies to all specified forks, and only needs to run once
```

Comment:
Any here is not the wording I was looking for. It's not an exclusive thing, it's a collective thing. There may be cases where you transition from one to fork to another, so it applies to all. I can see how "any" would work: maybe you want to hit the test when you're testing either one of the forks. I'll see if there's a cleaner way of describing this, or maybe remove it all together. It's completely up to the test-runner code to handle it in a way applicable to the thing being tested. For us it's just a statement on what's covered by a test, regardless of the way it's running.

Reply:
Made it more minimal, we can formalize pure semantics later. And it may even be up to the implementer of the test, as it doesn't change the test behavior: instead, it could just be a way of ensuring equal functionality with different forks "activated" (yet to see this in practice).
See d64a4f2

Reply:
@protolambda this one is good too, clean and understandable